tests/cypress/support/commands.js

```diff
@@ -395,3 +395,11 @@ Cypress.Commands.add('checkPolicyNoResourcesIconMessage', (present=true, message
 Cypress.Commands.add('checkNotificationMessage', (kind, title, notification) => {
   cy.then(() => action_checkNotificationMessage(kind, title, notification))
 })
+
+Cypress.Commands.add('doTableSearch', (text, inputSelector = null, parentSelector = null) => {
+  doTableSearch(text, inputSelector, parentSelector)
```

Comment:
@kkaarreell Should we keep the command and function name different as we are following for other commands 
```
Cypress.Commands.add('checkNotificationMessage', (kind, title, notification) => {
  cy.then(() => action_checkNotificationMessage(kind, title, notification))
})
Cypress.Commands.add('verifyClusterViolationsInListing', (clusterName, violationsCounter, violatedPolicies) => {
  cy.then(() => action_verifyClusterViolationsInListing(clusterName, violationsCounter, violatedPolicies))
})
```

Reply:
I think we can/should change it, but since it would require way more updates, I would prefer doing that in a different PR after commands are introduced.